src/main/java/net/dries007/tfc/ConfigTFC.java

```diff
@@ -307,6 +307,20 @@ public static void onConfigChangedEvent(ConfigChangedEvent.OnConfigChangedEvent
         @Config.Comment("The default length of a month (in days) when a new world is started. This can be changed in existing worlds via the /timetfc command.")
         @Config.LangKey("config." + MOD_ID + ".general.defaultMonthLength")
         public int defaultMonthLength = 8;
+
+        @Config.Comment("Should animals became old and die?")
+        @Config.LangKey("config." + MOD_ID + ".general.enableAnimalAging")
+        public boolean enableAnimalAging = true;
+
+        @Config.Comment("How long until animals became old (in factor to adulthood)?")
+        @Config.RangeDouble(min = 1, max = 50)
+        @Config.LangKey("config." + MOD_ID + ".general.factorAnimalAging")
+        public double factorAnimalAging = 3;
+
+        @Config.Comment("Chance of animal dying (checked every in-game day) after it became old")
+        @Config.RangeDouble(min = 0, max = 1)
+        @Config.LangKey("config." + MOD_ID + ".general.chanceAnimalDeath")
+        public double chanceAnimalDeath = 0.05;
```

Comment:
I think this should be set to 0 by default, I'm not entirely convinced it's a feature that won't just lead to excessive respawning and/or death of animals. Is there any visual indication of getting old?

Reply:
i think old animals should no longer be able to breed, but remove the option of them dying. atm theres no visual indication of them getting old as far as im aware

Reply:
Yes, no indication. I will make their textures location update via code and duplicate in the assets, so, in time, someone better than me can do something about it 😛